doc/integrator/extend_data_model.rst

```diff
@@ -1,55 +1,245 @@
-in the file ``<package>/forms.py``.
+The Data model can be extended in the file ``geoportal/<package>_geoportal/models.py``.
 
-For example to add some user details, including a link to a
-new class named "Title", add to ``<package>/models.py``::
+For example to add some user details, including a link to a new class named "Title",
+add to ``geoportal/<package>_geoportal/models.py``:
+
+.. code:: python
 
     # Used to hide the original user in the admin interface
     User.__acl__ = [DENY_ALL]
 
     class UserDetail(User):
-        __label__ = _('userdetail')
-        __plural__ = _('userdetails')
         __tablename__ = 'userdetail'
         __table_args__ = {'schema': _schema}
         __acl__ = [
             (Allow, Authenticated, ALL_PERMISSIONS),
         ]
         __mapper_args__ = {'polymorphic_identity': 'detailed'}
-        id = Column(types.Integer, ForeignKey(_schema + '.user.id'),
-                primary_key=True)
+        __colanderalchemy_config__ = {
+            'title': _('User detail'),
+            'plural': _('User details')
+        }
+        id = Column(
+            types.Integer,
+            ForeignKey(_schema + '.user.id'),
+            primary_key=True
+        )
 
-        phone = Column(types.Unicode, nullable=False, label=_(u'phone'))
+        phone = Column(
+            types.Unicode,
+            nullable=False,
+            info={
+                'colanderalchemy': {
+                    'title': _('Phone')
+                }
+            }
+        )
 
         # title
         title_id = Column(Integer, ForeignKey(_schema + '.title.id'), nullable=False)
-        title = relationship("Title", backref=backref('users'))
-
-        def __init__(self, username=u'', password=u'', functionalities=[],
-                     phone=u'', email=u'', title=None, role=None):
-            User.__init__(self, username, password, email, functionalities, role)
-            self.phone = phone
-            self.title = title
+        title = relationship("Title", backref=backref('users'), info={
+            'colanderalchemy': {
+                'title': _('Title')
+            }
+        })
 
     class Title(Base):
-        __label__ = _('title')
-        __plural__ = _('titles')
         __tablename__ = 'title'
         __table_args__ = {'schema': _schema}
         __acl__ = [
             (Allow, Authenticated, ALL_PERMISSIONS),
         ]
+        __colanderalchemy_config__ = {
+            'title': _('Title'),
+            'plural': _('Titles')
+        }
         id = Column(types.Integer, primary_key=True)
-        name = Column(types.Unicode, nullable=False, label=_(u'Name'))
-        description = Column(types.Unicode, label=_(u'Description'))
+        name = Column(types.Unicode, nullable=False, info={
+            'colanderalchemy': {
+                'title': _('Name')
+            }
+        })
+
+And for the admin interface add create the following empty files:
```

Comment:
Suggestion: "Now you need to extend the administration user interface. For this, first ensure that the following files exist (if needed, create them as empty files):"